numpy/lib/tests/test_io.py

```diff
@@ -480,6 +480,14 @@ def test_record(self):
         y = np.loadtxt(d, dtype=mydescriptor)
         assert_array_equal(y, b)
 
+    def test_tab_delimited_spaces(self):
+        "Test using an explicit dtype with a line of tab delimited spaces"
+        data = StringIO("aa\tbb\n \t \ncc\tdd")
```

Comment:
Universal failure because StringIO is not there. By the way, we prefer to make the first line a comment (with #), nose prints the line otherwise in verbose mode if it is a doc string.